docs/contributing-to-airbyte/writing-docs.md

```diff
@@ -13,7 +13,7 @@ The Docs team maintains a list of [#good-first-issues](https://github.com/airbyt
 
 ## Contributing to Airbyte docs
 
-Before contributing to Airbyte docs, read the Airbyte Community [Code of Conduct](code-of-conduct.md)
+Before contributing to Airbyte docs, read the Airbyte Community
```

Comment:
Looks like this link accidentally got deleted instead of updated (see screenshot from prod):
![Screenshot 2023-10-26 at 3 05 07 PM](https://github.com/airbytehq/airbyte/assets/7516653/cdae3391-c249-4e4b-ba70-4ace58550c6f)

I noticed the issue while working on some new edits to this file, so I'll just tack on another commit replacing the updated link 👍